packages/hint/src/lib/types/jsdom-async-html.ts

```diff
@@ -1,4 +1,5 @@
-import { IAsyncHTMLDocument, IAsyncHTMLElement } from './async-html'; //eslint-disable-line
+import { IAsyncHTMLDocument, IAsyncHTMLElement, IAsyncWindow } from './async-html'; //eslint-disable-line
```

Comment:
What does it happen if you remove `//eslint-disable-line`?

Reply:
Nothing it seems. Want me to remove it? I was originally only making necessary changes (as this comment was already in the file), but it's an easy change so I can include the removal in this PR.

Reply:
Yes please. I saw it wasn't your change, but it would be nice to remove it :)